.env.example

```diff
@@ -27,6 +27,13 @@ OPTIMIZE_CLIENT_ID=fill-this-in
 OPTIMIZE_CLIENT_SECRET=fill-this-in
 ### This value comes from the `CAMUNDA_OPTIMIZE_BASE_URL`.
 OPTIMIZE_BASE_URL=fill-this-in
-
 ## This value will only change if you're not using SaaS.
-OPTIMIZE_AUDIENCE=optimize.camunda.io
\ No newline at end of file
+OPTIMIZE_AUDIENCE=optimize.camunda.io
+
+# Required for Web Modeler API tutorial, to access API
+MODELER_BASE_URL=https://modeler.cloud.camunda.io/api/v1
+MODELER_AUDIENCE=modeler.cloud.camunda.io
+MODELER_CLIENT_ID=fill-this-in
+MODELER_CLIENT_SECRET=fill-this-in
```

Comment:
Please be consistent here and follow the same pattern set by the Optimize and Admin APIs.

```
# Web Modeler API tutorial
```

Then, for each value the user has to fill in, provide a comment about where this value comes from unless this is exactly how it appears on the UI. I got the following when creating my org level credentials with Modeler Scopes. 

```
export CAMUNDA_CONSOLE_CLIENT_ID='xxx'
export CAMUNDA_CONSOLE_CLIENT_SECRET='xxx'
export CAMUNDA_OAUTH_URL='https://login.cloud.camunda.io/oauth/token'
export CAMUNDA_CONSOLE_BASE_URL='https://api.cloud.camunda.io'
export CAMUNDA_CONSOLE_OAUTH_AUDIENCE='api.cloud.camunda.io'
```

Reply:
Adjusted in latest commit 👍